### DISCUSSION TITLE
Desktop App: Has no ability to link self hosted

### DISCUSSION THREAD

**Discussion by compgeniuses** [toeverything/AFFiNE#8261] [Ideas]
the desktop app, can sync with hosted cloud version, and has no ability to link with a self hosted instance of affine cloud

**Comment by richie80** [comment]
Is fixed since version 19.0

**Comment by hannah23280** [comment]
May i ask, if i am using the desktop app, i can use all its functionalities without linking to another serve.
Hence what is the advantage gained by linking it to a self-hosted affine application?
